Extensions/DialogueTree/dialoguetools.js

```diff
@@ -472,7 +472,7 @@ gdjs.dialogueTree.goToNextDialogueLine = function() {
       this.dialogueText = this.dialogueData.text;
       this.commandCalls = [];
     } else {
-      this.dialogueText += this.dialogueData.text;
+      this.dialogueText += ' ' + this.dialogueData.text;
```

Comment:
What should happen in these cases:

* `my<<command>>text` => `mytext` I guess?
* `my <<command>> text` => ??
* `<<command>>text` => ?? (notice the command is the very first thing.

It seems that your solution is giving `my text` in the second case (which is fine?) (but please check ;)), but ` text` in the third case, while this should be `text`?

Reply:
`my <<command>> text` => `my  text ` two->two spaces as expected

but if I do `my<<command>>text` , it adds a space but I kind of like the fact that it adds the space for the user if they missed it

Reply:
I need to look into this more carefully :) 
Ideally it should not add or remove a space, but keep everything. Its interesting that it does it only to spaces before or after a command or an option type

Reply:
Ok I updated the PR to treat the case with continuing the text after a choice is taken and the node after starts with a command. It wasnt restaring the scrolling

Reply:
Yeah not a big fan of adding a space with commands 😬
What about?
```suggestion
      this.dialogueText += (this.dialogueText === '' ? '' : ' ') + this.dialogueData.text;
```

i.e: add a space only if the text is not empty? (not tested)

Reply:
nah, it still adds spaces. Let me see if I can better fix this

Reply:
@4ian I am now convinced that this is a bug in bondagejs. Some regular expression in there is trimming spaces.
Your suggested fix makes the alleviates the problem considerably and makes it appear very rarely, but doesnt completely fix it.

I suggest to use this workaround fix for now, at least I manage to get a propper one merged in bondagejs